modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/TypeScriptAngularClientCodegen.java

```diff
@@ -246,6 +246,8 @@ public void postProcessParameter(CodegenParameter parameter) {
             // Prep a string buffer where we're going to set up our new version of the string.
             StringBuilder pathBuffer = new StringBuilder();
             StringBuilder parameterName = new StringBuilder();
+            /** Whether to skip characters inside curly braces */
+            Boolean skipThisAndSubsequentChars = false;
```

Comment:
Maybe use the primitive `boolean` type here and change the comment to the actual function of this variable?
Below it's not only used to skip chars in curly braces, but also to skip chars after a colon.

Reply:
@TiFu done, I updated the commend and changed the type to `boolean`